src/ensembl/production/metadata/api.py

```diff
@@ -12,26 +12,35 @@
 import sqlalchemy as db
 from sqlalchemy.orm import Session
 import pymysql
-
+import models
```

Comment:
test case failed as models module not found either you can use relative import  or `import  enseml.production.metadata.models`